tests/test-stack-trace.js

```diff
@@ -124,6 +136,14 @@ var testDefs = [
     limit: 1
   },
 
+  { name: 'top most webppl entry is not a user function',
+    code: ['var f = function() { stackTraceTestFns.test() };',
```

Comment:
I don't follow how this test works. `stackTraceTestFns.test` is called as a method, so -- due to the way we wrap external functions -- I expect `s, k, a` above to be all `undefined`, and the inner function in `test` never to be called. Is that expected behavior, or am I missing something?